tests/test_crud.py

```diff
@@ -8,6 +8,7 @@
 
 IGNORED_WARNINGS = [
     "Activation Key 'Test Activation Key Copy' already exists.",
+    "You have configured a plain HTTP server URL. All communication will happen unencrypted.",
```

Comment:
Now this is an "expected warning"...

Reply:
We use HTTP in the inventory tests, and I am not going to fix that ;)